### ISSUE TITLE
[BUG] A11y | Accordion is not accessible

### ISSUE THREAD

**Issue by pixelass** [DivanteLtd/storefront-ui#587]
**Describe the bug**
The accordion are not accessible.  
They can't be focused and don't allow interaction via keyboard input.  
There is no focus style.

**How to reproduce**
1. Go to https://storybook.storefrontui.io/?path=/story/organisms-accordion--common
2. Click on any accordion
3. Try to use the keyboard to interact with the accordion
4. The accordion does not allow interaction via keyboard

**Expected behavior**
I should be able to interact with the accordion via keyboard commands

**Actual behavior**
The accordion can not be opened, closed, navigated or focused via keyboard

**Caused by**

The accordion is not build with accessibility in mind. 

The usage of underlying checkboxes or `tabindex` with the correct event handlers can fix this issue.

**Comment by peoray** [comment]
Can I work on this issue if no one is on it yet?

**Comment by aniamusial** [comment]
@peoray sure, just remember to self-assign - thank you!

**Comment by aniamusial** [comment]
@peoray how is it going? any blockers? :)

**Comment by peoray** [comment]
@aniamusial I haven't actually started working on this, but good thing you commented, getting to work immediately ;)

**Comment by aniamusial** [comment]
Closing as its fixed in #588
